pkg/subctl/operator/install/deployment/ensure.go

```diff
@@ -41,19 +40,57 @@ func Ensure(restConfig *rest.Config, namespace string, image string) (bool, erro
 		return false, err
 	}
 
-	deployment := &appsv1.Deployment{}
-
-	err = embeddedyamls.GetObject(embeddedyamls.Operator_yaml, &deployment)
-
-	deployment.Spec.Template.Spec.Containers[0].Image = image
-
+	replicas := int32(1)
+	imagePullPolicy := v1.PullAlways
 	// If we are running with a local development image, don't try to pull from registry
-	if image == "submariner-operator:local" {
```

Comment:
Oh  I though this was the reason, what's the advantage of converting it to explicit?

Reply:
I thought it would be interesting to have a single function for all operators (Submariner and KubeFed).